objectherkenning_openbare_ruimte/detection_pipeline/run_detection_pipeline.py

```diff
@@ -0,0 +1,21 @@
+import time
+
+from objectherkenning_openbare_ruimte.detection_pipeline.components.data_detection import (
+    DataDetection,
+)
+from objectherkenning_openbare_ruimte.settings.settings import (
```

Comment:
does this work? I see no settings/settings path

Reply:
Yes, the settings path thing is only for AML because of how AML executes the jobs. Outside it's not needed.